tests/TestCase/Serializer/RequestTest.php

```diff
@@ -39,6 +39,13 @@ class RequestTest
     public function addValidation(Validator|array $validator): Validator
     {
         $validator->requirePresence('fieldInt', true, 'Не указан fieldInt');
+        if (!empty($this->fieldObject)) {
+            $validator->addNested('fieldObject', $this->fieldObject->addValidation(new Validator()));
+        }
+
+        if (!empty($this->objects)) {
+            $validator->addNestedMany('objects', $this->objects[0]->addValidation(new Validator()));
+        }
```

Comment:
Проблема в этих строках.

**Варианта исправления 2**

**Вариант 1** - ломает валидатор из-за невозможности преобразовать `int` к `string` в файле `vendor/cakephp/cakephp/src/Validation/Validator.php:274`:

````suggestion
        $objectsValidator = new Validator;
        foreach ($this->objects as $key => $object) {
            $objectsValidator->addNested((string)$key, $object->addValidation(new Validator));
        }
        $validator->addNested('objects', $objectsValidator);
````


**Вариант 2** - рабочий - добавляем один и тот же валидатор в себя:
````suggestion
        $validator->addNestedMany('objects', $validator);
````